tests/pytorch/test_datasets_pt.py

```diff
@@ -211,11 +211,13 @@ def test_charactergenerator():
     ],
 )
 def test_mock_dataset(dataset_name, train, input_size, size, rotate, mock_svhn_dataset):
+    # TODO: how to handle new path to temporary mock dataset?
+
     ds = datasets.__dict__[dataset_name](
         train=train, download=False, sample_transforms=Resize(input_size), rotated_bbox=rotate,
     )
```

Comment:
@fg-mindee
back to a work in progress sweat_smile
Do you have any recommendations to get the mocked dataset temp path into `_VisionDataset ` archive_path ?
Otherwise, suggestions are also very welcome for each other change